binary_transparency/firmware/README.md

```diff
@@ -339,3 +339,29 @@ go run cmd/publisher/publish.go --logtostderr --v=2 --timestamp="2020-10-10T23:0
 >
 > Anybody else running a monitor also knows that malicious firmware has been
 > logged and can raise the alarm.
+
+Generating a Verifiable Map
+---------------------------
+
```

Comment:
Maybe we should more strongly flag this section as researchy/WIP for the moment?